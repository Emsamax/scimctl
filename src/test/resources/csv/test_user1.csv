userName,name.familyName,name.givenName.value,name.givenName.shortName,name.honorificPrefix,displayName,*emails,*phoneNumbers,active,*roles
jdoe,Doe,John,J, M.,John Doe,"value=jdoe@example.com,type=work,primary=true;value=jdoe2@example.com,type=personal,primary=false","value=+12345678901,type=mobile",true,"value=user,display=user;value=User,display=User"