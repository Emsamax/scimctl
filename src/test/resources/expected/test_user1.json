[
  {
    "userName": "jdoe",
    "name": {
      "familyName": "Doe",
      "givenName": {
        "value": "John",
        "shortName": "J"
      },
      "honorificPrefix": "M."
    },
    "displayName": "John Doe",
    "emails": [
      {
        "value": "jdoe@example.com",
        "type": "work",
        "primary": "true"
      },
      {
        "value": "jdoe2@example.com",
        "type": "personal",
        "primary": "false"
      }
    ],
    "phoneNumbers": [
      {
        "value": "+12345678901",
        "type": "mobile"
      }
    ],
    "active":"true",
    "roles": [
      {
        "value": "user",
        "display": "user"
      },
      {
        "value": "User",
        "display": "User"
      }
    ]
  }
]